src/plugins.ts

```diff
@@ -366,4 +348,22 @@ export default class Plugins {
     )
     return plugins
   }
+
+  private isValidPlugin(p: Config): boolean {
+    if (p.valid) return true
```

Comment:
This is set here to differentiate between `oclif/core` and `cli-engine` plugins:
https://github.com/oclif/core/blob/a18803454449986e03549838ba3928c66f98da37/src/config/plugin.ts#L161-L165